client/components/admin/admin-pages-edit.vue

```diff
@@ -165,7 +187,9 @@
 <script>
 import _ from 'lodash'
 import { StatusIndicator } from 'vue-status-indicator'
+import gql from 'graphql-tag'
 
+import localesQuery from 'gql/admin/locale/locale-query-list.gql'
```

Comment:
You actually don't need to remotely fetch the installed locales, you have them already in the `siteLangs` variable, which is available everywhere.

Example value:
```
[
  {
    "code":"en",
    "name":"English"
  },
  {
    "code":"fr",
    "name":"français"
  }
]
```

Reply:
Oh ok, i will fix that :)